src/main/resources/com/arcbees/bourseje/client/resources/css/pages/candidates.gss

```diff
@@ -0,0 +1,42 @@
+.text_intro {
+    margin-bottom: 100px;
+}
+
+.text_intro h2,
+.text_intro p {
+    font-size: 3rem;
+    line-height: 1.2;
+}
+
+.text_intro p span {
+    display: block;
+}
+
+.box_candidates {
+    background-color: C_SECONDARY;
+
+    padding: 20px 10px 0;
+    margin-bottom: 10px;
+}
+
+.box_candidates h3 {
+    font-size: 3rem;
+    font-family: "brandonGrotesqueBold";
```

Comment:
You could create a Fonts.java and add this as a variable like F_PRIMARY ... much easier to remember. I would also add fallback fonts, like Arial, sans-serif. Adding another java file, you will probably want to use something like a theme. Hey guess what! There's a brand new blogpost about that : http://blog.arcbees.com/2015/04/28/managing-your-css-files-with-variables-and-a-theme/ ... okay you already know how to do that but hey, I need to promote it!